src/test/java/uk/gov/hmcts/reform/userprofileapi/service/impl/ValidationHelperServiceTest.java

```diff
@@ -117,72 +107,76 @@ public void testValidateUpdateUserProfileRequestValidHappyPath() {
 
     @Test(expected = RequiredFieldMissingException.class)
     public void testValidateUpdateUserProfileRequestValidException() {
-        doThrow(RequiredFieldMissingException.class).when(exceptionServiceMock).throwCustomRuntimeException(eq(ExceptionType.REQUIREDFIELDMISSINGEXCEPTION), any(String.class));
-
         updateUserProfileData.setIdamStatus(null);
 
         sut.validateUpdateUserProfileRequestValid(updateUserProfileData, "f56e5539-a8f7-4ae6-b378-cc1015b72dcc", API);
     }
 
     @Test
-    public void testValidateUpdateUserProfileRequestValidPersistAuditOnException() {
-        doNothing().when(exceptionServiceMock).throwCustomRuntimeException(any(ExceptionType.class), any(String.class));
-
+    public void test_ValidateUpdateUserProfileRequestValid_PersistAuditOnException() {
         updateUserProfileData.setIdamStatus(null);
 
-        sut.validateUpdateUserProfileRequestValid(updateUserProfileData, "f56e5539-a8f7-4ae6-b378-cc1015b72dcc", API);
+        final Throwable raisedException = catchThrowable(() -> sut.validateUpdateUserProfileRequestValid(updateUserProfileData, "f56e5539-a8f7-4ae6-b378-cc1015b72dcc", API));
+        assertThat(raisedException).isInstanceOf(RequiredFieldMissingException.class);
 
         verify(auditServiceMock, times(1)).persistAudit(eq(HttpStatus.BAD_REQUEST), eq(API));
     }
 
     @Test(expected = Test.None.class)
     public void testvalidateUserStatusBeforeUpdate_scenario1() {
-        sut.validateUserStatusBeforeUpdate(updateUserProfileData, userProfile, API);
+        final Throwable raisedException = catchThrowable(() -> sut.validateUserStatusBeforeUpdate(updateUserProfileData, userProfile, API));
+
+        assertThat(raisedException.getMessage()).contains("User is PENDING or input status is PENDING and only be changed to ACTIVE or SUSPENDED for userId: null");
+        assertThat(raisedException).isInstanceOf(RequiredFieldMissingException.class);
 
         verify(auditServiceMock, times(1)).persistAudit(eq(HttpStatus.BAD_REQUEST), eq(API));
     }
 
     @Test(expected = Test.None.class)
```

Comment:
expected should be removed

Reply:
This has been addressed in the following commit: https://github.com/hmcts/rd-user-profile-api/pull/333/commits/3e3d5a6b05b395c1e8271f85b7e7c1f485182407